src/index.ts

```diff
@@ -419,6 +419,17 @@ class QueryPipeline extends EventEmitter {
         return obj;
       }, {});
 
+    let activePins = new Set();
```

Comment:
A set won't work anymore. But we can save a bit of time by using an object which contains sets.
```
const activePins: {
  [fieldName: string]: Set<string>,
} = {};
```